### ISSUE TITLE
Blank input when selecting drug causes crash

### ISSUE THREAD

**Issue by nicktimko** [M4cs/drugwars#1]
# Steps to Reproduce

1. Launch
2. Enter to play
3. B to buy
4. Hit enter without entering anything

```
Traceback (most recent call last):
  File "/home/nick/.pipx/bin/drugwars", line 8, in <module>
    sys.exit(main())
  File "/home/nick/.pipx/venv/venvs/drugwars/lib/python3.8/site-packages/drugwars/__main__.py", line 32, in main
    display_pricing_screen(p)
  File "/home/nick/.pipx/venv/venvs/drugwars/lib/python3.8/site-packages/drugwars/events.py", line 423, in display_pricing_screen
    if not check_drug_inp(buy):
  File "/home/nick/.pipx/venv/venvs/drugwars/lib/python3.8/site-packages/drugwars/helpers.py", line 12, in check_drug_inp
    if a[0].lower() == "c":
IndexError: string index out of range
```

Most other prompts tolerate missing input

**Comment by M4cs** [comment]
Hi sorry for the late response, this has been fixed in v1.1.7 :)

**Comment by M4cs** [comment]
Thank you for reporting this bug to me!
